test/ReflectorTest.php

```diff
@@ -189,18 +191,20 @@ function ( FunctionEntity $entity ) {
 	    $this->assertEquals( $expectedMethods, $functionNames );
     }
 
-	public function regexFiltersAndExpectedMethods() {
+	public function regexFiltersAndExpectedMethods()
+    {
```

Comment:
It still has the white space problem. Please consider original [code](https://github.com/victorjonsson/PHP-Markdown-Documentation-Generator/blob/master/src/PHPDocsMD/ClassEntity.php).

Reply:
This piece of code, seen in an editor and in the files list (https://github.com/victorjonsson/PHP-Markdown-Documentation-Generator/pull/19/files#diff-7d7210c1807e215172a654507390aae1R195) is ok.
No idea why it shows "weird" in the diff.